openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java

```diff
@@ -142,6 +142,14 @@ private boolean validateNotEmpty(EditText editText) {
         return StringUtils.notEmpty(editText.getText().toString());
     }
 
```

Comment:
Can you add a Java doc comment that describes this method? E.g.
```suggestion
/**
   * <Method description here>
   * @return True if a location is selected or no location needed for the OpenMRS instance used.
   */
```